### PR TITLE
Ported some of the tests from TestCommandMore

### PR DESCRIPTION
**Pull request by Anipik**
Related to https://github.com/dotnet/machinelearning/issues/984
- Ported some of the tests from TestCommandMore. Remaining tests contain modules or datasets that are not yet available

cc @danmosemsft @eerhardt

### CODE REVIEW

File: test/Microsoft.ML.TestFramework/TestCommandMore.cs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+namespace Microsoft.ML.Runtime.RunTests
+{
+    public sealed partial class TestCommandMore : TestSteppedDmCommandBase
+    {
+        public TestCommandMore(ITestOutputHelper helper) : base(helper)
+        {
+        }
+
+        [Fact]
```
Comment (eerhardt) [inline test/Microsoft.ML.TestFramework/TestCommandMore.cs]: I don't think we should be adding more tests to the `TestFramework` library.  I think the `TestFramework` library should just have common code, and the tests should be in other libraries.

Comment (Anipik) [reply test/Microsoft.ML.TestFramework/TestCommandMore.cs]: what should be the correct place for these tests ?
```CommandCVLinearClassification ``` could be moved to predictors.cs
but what about 
```
CommandTrainScorePriorTrainer
CommandVersionTypeCheck
```

Comment (eerhardt) [reply test/Microsoft.ML.TestFramework/TestCommandMore.cs]: @TomFinley - thoughts?

**Comment by Anipik** [comment]
@TomFinley can you suggest an appropriate place to place these tests ?

**Comment by eerhardt** [comment]
How about `Microsoft.ML.Tests` for now? That is a better place than `TestFramework`. We can move them "somewhere better" if that "somewhere better" comes along.

**Pull request closed without merging**